projects/go-lib/src/lib/components/go-radio/go-radio-button.component.scss

```diff
@@ -5,32 +5,32 @@
 }
 
 .go-radio__custom-radio {
```

Comment:
changes to this file are only wrt having css properties alphabetically arranged.

Reply:
@Ashygoyal typically, we would try to avoid doing stuff like this if it's not directly related to the bug/feature being worked on so that it makes it clearer what is being changed.

I won't make you change it though, in this case. Regardless, thanks for looking out.

Reply:
Noted @grahamhency . I thought it wont make sense to raise a separate PR just for this small change. But I get your point. Next time onwards, a new PR would make more sense, in order to avoid confusions.